aemedge/styles/styles.css

```diff
@@ -226,14 +228,17 @@ a.button::-moz-focus-inner {
 a.button.primary,
 button.primary,
 a.button.secondary,
-button.secondary {
+button.secondary,
+a.button.steam,
+button.steam {
   border-radius: 1600px;
   opacity: 1;
   min-width: 11.25rem;
-  font-size: 1rem;
+  font-size: 1.125rem;
   padding: 0.875rem 1rem;
   min-height: 60px;
   align-content: center;
+  font-weight: 800;
```

Comment:
I think you can use the font-size and font-weight changes you created on the "button.secondary" class, instead of creating the new "steam" style.